hyper/http11/connection.py

```diff
@@ -130,6 +133,15 @@ def request(self, method, url, body=None, headers={}):
         if self._sock is None:
             self.connect()
 
+        #add HTTP Upgrade headers
+        headers[b'connection'] = b'Upgrade, HTTP2-Settings'
+        headers[b'upgrade'] = H2C_PROTOCOL
+        
+        #need to encode SETTINGS frame payload in Base64 and put into the HTTP-2 Settings header 
+        http2_settings = SettingsFrame(0)
+        http2_settings.settings[SettingsFrame.INITIAL_WINDOW_SIZE] = 65535
+        headers[b'HTTP2-Settings'] = base64.b64encode(http2_settings.serialize_body())
```

Comment:
Sending these all the time is wasteful, and I agree just the first request on a connection should be sufficient.